src/Apps/CollectorProfile/Routes/Saves/Components/EditListPrivacyModal/EditListPrivacyModal.tsx

```diff
@@ -0,0 +1,150 @@
+
+const logger = createLogger("EditListPrivacyModal")
+
+export const EditListPrivacyModal: React.FC<EditListPrivacyModalProps> = ({
+  onClose,
+  me,
+}) => {
+  const { t } = useTranslation()
+  const customArtworkLists = extractNodes(me.customArtworkLists)
+  const savedArtworksArtworkList = me?.savedArtworksArtworkList
+  const artworkLists = savedArtworksArtworkList
+    ? [savedArtworksArtworkList, ...customArtworkLists]
+    : customArtworkLists
+  const initialValues = artworkLists && getInitialValues(artworkLists)
+
+  const { submitMutation } = useUpdateMeCollection()
+
+  const { sendToast } = useToasts()
+
+  const handleSubmit = async (formikValues: EditListPrivacyFormikValues) => {
+    try {
+      await submitMutation({
+        variables: {
+          input: {
+            attributes: (Object.entries(formikValues).map(
+              ([id, shareableWithPartners]) => ({
+                id,
+                shareableWithPartners,
+              })
+            ) as unknown) as UpdateMeCollectionInput[],
+          },
+        },
+        rejectIf: res => {
+          const result = res.updateMeCollectionsMutation?.meCollectionsOrErrors
+
+          return result && result[0].mutationError
+        },
+      })
+
+      sendToast({
+        variant: "success",
+        message: t("collectorSaves.editListPrivacyModal.success"),
+      })
+
+      onClose()
+    } catch (error) {
+      logger.error(error)
+    }
+  }
+
+  return (
+    <ModalDialog
+      width={["100%", 600]}
+      onClose={onClose}
+      title={t("collectorSaves.editListPrivacyModal.title")}
+      data-testid="EditListPrivacyModal"
+    >
+      <Formik<EditListPrivacyFormikValues>
+        initialValues={initialValues}
+        onSubmit={handleSubmit}
+      >
+        {formik => (
+          <Form>
+            <Text variant="xs">
+              {t("collectorSaves.editListPrivacyModal.description")}{" "}
+              <Link
+                href="https://support.artsy.net/s/article/Offers-on-saved-works"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                Learn more
+              </Link>
+            </Text>
+
+            <Spacer y={2} />
+
+            <Flex
+              flexDirection="column"
+              overflowY="scroll"
+              height="320px"
+              paddingTop={1}
+            >
+              <Join separator={<Spacer y={2} />}>
+                {artworkLists.map(list => {
+                  return (
+                    <EditArtworkListItemFragmentContainer
+                      key={list.internalID}
+                      item={list}
+                    />
+                  )
+                })}
+              </Join>
+            </Flex>
+
+            <Spacer y={2} />
+
+            <Flex justifyContent="flex-end">
+              <Button
+                type="submit"
+                loading={!!formik.isSubmitting}
```

Comment:
#non-blocking: The loading state may take some time if collector is updating a large number of lists in a batch. The button shows a loading spinner and I wonder what state the toggles should be when the form is submitting. I guess collector can still toggle things while submitting the form. 🤔 

I don't think it will create errors or anything, and from the recording updating lists in a batch seems to be fast enough.